src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import duke.exception.WriteToStorageException;
+import duke.task.Task;
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskListTest {
+
+    @Test
+    void getStore() {
+        //TODO
+    }
+    @Test
+    void addTask() throws WriteToStorageException {
+        Task task = new Todo("hello world");
+        TaskList tasks =  new TaskList(new StorageStub());
```

Comment:
Is there more than one whitespace between the "=" and "new"? Noticed the same issue below as well.